kong.config.sh

```diff
@@ -159,3 +159,30 @@ PAYLOAD
 }
 PAYLOAD
 # no auth: used for middleware <-> device communication via HTTP(s)
+
+
+# CA certificate retrievemment
+(curl -o /dev/null $kong/apis -sS -X POST \
+    --header "Content-Type: application/json" \
+    -d @- ) <<PAYLOAD
+{
+    "name": "ejbca-ca",
+    "uris": "/ca",
+    "strip_uri": true,
+    "upstream_url": "http://ejbca:5583/ca/"
+}
+PAYLOAD
+authConfig "ejbca-ca"
+
+# certificate sign requests
+(curl -o /dev/null $kong/apis -sS -X POST \
+    --header "Content-Type: application/json" \
+    -d @- ) <<PAYLOAD
+{
+    "name": "ejbca-certs",
+    "uris": "/sign",
+    "strip_uri": true,
+    "upstream_url": "http://ejbca:5583/sign/"
+}
+PAYLOAD
+authConfig "ejbca-certs"
```

Comment:
Regarding the entire addition for at this file, to avoid two posts and centralize the ejbca configuration, maybe something like this:
```json
{
     "name": "ejbca-paths",
     "uris": [ "/sign", "/ca"],
     "strip_uri": false,
     "upstream_url": "http://ejbca:5583/"
 }
```

would be simpler, as there is no need to strip the uri .

What do you guys think @alexandre-vasc and @giovannicuriel

Reply:
👍

Reply:
I was not aware of this syntax. Its a good idea!